src/routes/categories/containers/CategoriesToolbar.js

```diff
@@ -9,31 +9,55 @@ import ToolbarItem from '../../../modules/layout/components/ToolbarItem'
 import { apiUrl } from '../../../modules/endpoint/constants'
 import CategoryModel from '../../../modules/endpoint/models/CategoryModel'
 import type { TFunction } from 'react-i18next'
+import FeedbackModal from '../../feedback/components/FeedbackModal'
+import CityModel from '../../../modules/endpoint/models/CityModel'
+import { POSITIVE_RATING } from '../../../modules/endpoint/FeedbackEndpoint'
+import styled from 'styled-components'
+import FeedbackLink from '../../feedback/components/FeedbackLink'
+import type { LocationState } from 'redux-first-router'
+
+const FeedbackToolbarItem = styled(FeedbackLink)`
+  display: inline-block;
+  margin: 0 10px;
+  padding: 8px;
+`
 
 type PropsType = {
-  city: string,
-  language: string,
+  cities: Array<CityModel>,
   categories: CategoriesMapModel,
-  pathname: string,
+  location: LocationState,
   t: TFunction
 }
 
 export class CategoriesToolbar extends React.PureComponent<PropsType> {
   getPdfUrl (category: CategoryModel): string {
+    const {city, language} = this.props.location.payload
+
     if (category.id === 0) {
-      return `${apiUrl}/${this.props.city}/${this.props.language}/wp-json/ig-mpdf/v1/pdf`
+      return `${apiUrl}/${city}/${language}/wp-json/ig-mpdf/v1/pdf`
     } else {
-      return `${apiUrl}/${this.props.city}/${this.props.language}/wp-json/ig-mpdf/v1/pdf?url=${category.path}`
+      return `${apiUrl}/${city}/${language}/wp-json/ig-mpdf/v1/pdf?url=${category.path}`
     }
   }
 
   render () {
-    const category = this.props.categories.findCategoryByPath(this.props.pathname)
+    const {t, cities, location} = this.props
+    const feedbackType = location.query && location.query.feedback
+    const category = this.props.categories.findCategoryByPath(location.pathname)
     if (!category) {
       return null
     }
     return <Toolbar>
-      <ToolbarItem name='file-pdf-o' text={this.props.t('createPdf')} href={this.getPdfUrl(category)} />
+      <ToolbarItem name='file-pdf-o' text={t('createPdf')} href={this.getPdfUrl(category)} />
+      <FeedbackToolbarItem isPositiveRatingLink location={location} />
+      <FeedbackToolbarItem isPositiveRatingLink={false} location={location} />
```

Comment:
The width is the same, only the height differs, but I think this okay because the height of a document is bigger than its width, whereas the width and height of a smiley should be the same. Or did I misunderstood you? But I could also force them to be of the same size of course.

Reply:
If you look closely, the "margin" of the icons vary (from `pdf` to `:)` there's less margin than from `:)` to `:(` )

Reply:
Ahh right, I just looked at the size. Fixed it.